app/controllers/admin/field_groups_controller.rb

```diff
@@ -64,10 +64,10 @@ def sort
     field_group_ids = params["#{asset}_field_groups"]
 
     field_group_ids.each_with_index do |id, index|
-      FieldGroup.update_all({:position => index+1}, {:id => id})
+      FieldGroup.update_all({position: index+1}, {id: id})
```

Comment:
Surrounding space missing for operator '+'.<br>Space inside { missing.<br>Space inside } missing.